packages/auth/src/resourceSet/service.ts

```diff
@@ -0,0 +1,67 @@
+import { AccessRequest } from '../access/types'
+import { JWKWithRequired } from '../client/service'
+
+import { BaseService } from '../shared/baseService'
+import { AccessService } from '../access/service'
+import { ResourceSet } from './model'
+
+interface ServiceDependencies extends BaseService {
+  accessService: AccessService
+}
+
+interface ResourceSetRequest {
+  access?: AccessRequest[]
```

Comment:
`access` is technically `REQUIRED` in the resource set registration request
https://datatracker.ietf.org/doc/html/draft-ietf-gnap-resource-servers#section-3.4